### PR TITLE
feat: [#185543521] remove landing page redirect to webflow

### PR DESCRIPTION
**Pull request by somabadri**
## Description

Removed the code that handles the redirect to the webflow landing page.

### Ticket

[#185543521](https://www.pivotaltracker.com/story/show/185543521)

### Notes

Follow up: remove the following environment variables in Circle Ci: `FEATURE_LANDING_PAGE_REDIRECT` and  `ALTERNATE_LANDING_PAGE_URL`

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation, if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config

### CODE REVIEW

File: web/src/pages/index.tsx [hunk 1]
```diff
@@ -75,9 +75,6 @@ const Home = (props: Props): ReactElement => {
     };
   }
   useEffect(() => {
-    const landingPageAlternateUrl = process.env.ALTERNATE_LANDING_PAGE_URL || ROUTES.landing;
```
Comment (FarazA22) [inline web/src/pages/index.tsx:78]: [Pebble] I think we should remove the environment variables from the .env-template. I think we can also remove it from bit warden, and the environment variables that are saved in circles ci

File: web/src/pages/index.tsx [hunk 2]
```diff
@@ -75,9 +75,6 @@ const Home = (props: Props): ReactElement => {
...
-    const alternateLandingPageEnabled = process.env.FEATURE_LANDING_PAGE_REDIRECT === "true";
```
Comment (FarazA22) [inline web/src/pages/index.tsx:79]: Same comment as above

File: web/src/pages/index.tsx (hunk 1 above)
Comment (somabadri) [reply web/src/pages/index.tsx:78]: yep, made a note of it above in the pr comment.

File: web/src/pages/index.tsx [hunk 3]
```diff
@@ -86,12 +83,6 @@ const Home = (props: Props): ReactElement => {
       } else if (userData === undefined && error !== undefined) {
```
Comment (FarazA22) [inline web/src/pages/index.tsx:83]: [Dust] Should we remove the else if conditional here and just keep what is inside of the code block?

Comment (FarazA22) [inline web/src/pages/index.tsx:83]: [Dust] Is the `else if` conditional necessary? Can we keep what is inside the code block?

Comment (somabadri) [reply web/src/pages/index.tsx:83]: The tests still pass without it but I like that it is explicitly spelled out, it makes it easier to read.

File: web/src/pages/index.tsx (hunk 1 above)
Comment (FarazA22) [reply web/src/pages/index.tsx:78]: The changes to .env-template should be part of this PR

Comment (somabadri) [reply web/src/pages/index.tsx:78]: They are not there by default in .env-template file.

**Pull request merged**